pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.poetry]
+name = "django-ses"
+version = "2.6.2"
+description = "A Django email backend for Amazon's Simple Email Service"
+authors = [
+    "Harry Marr <harry@hmarr.com>",
+    "Wes Winham <winhamwr@gmail.com>",
+    "Ross Lawley <ross.lawley@gmail.com>",
+    "Paul Craciunoiu",
+]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+    'Framework :: Django',
+    'Framework :: Django :: 2.2',
+    'Framework :: Django :: 3.2',
+    'Framework :: Django :: 4.0',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+]
+license = "MIT"
+readme = "README.rst"
+repository = "https://github.com/django-ses/django-ses"
+include = ["example", "tests"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+boto3 = ">=1.0.0"
+pytz = ">=2016.10"
+django = ">=2.2"
+importlib-metadata = {version = "^1.0", python = "<3.8"}
+requests = {version = "^2", extras = ["bounce", "events"]}
+M2Crypto = {version = "*", extras = ["bounce", "events"]}
```

Comment:
I think the correct way to do extras in poetry is like this instead:

```
[tool.poetry.extras]
bounce =  ["requests<3", "M2Crypto"]
events = ["requests<3", "M2Crypto"]
```

The way these extras are installed would be the same, `pip install django-ses[bounce,events]`
https://python-poetry.org/docs/pyproject/#extras

Reply:
Great. This is a bit confusing because poetry has an `--extras` flag you can pass to `poetry install` that generates it the way I had it, but the flag isn't well documented. 

In any case, this is tweaked.